README.md

```diff
@@ -14,11 +14,11 @@ Looking for typescript? Check out [graphqlgen](https://github.com/prisma/graphql
 | Boilerplate | less | more | more | some |
 | Docs | [docs](https://gqlgen.com) & [examples](https://github.com/99designs/gqlgen/tree/master/example) | [examples](https://github.com/graph-gophers/graphql-go/tree/master/example/starwars) | [examples](https://github.com/graphql-go/graphql/tree/master/examples) | [examples](https://github.com/samsarahq/thunder/tree/master/example)|
 | Query | :+1: | :+1: | :+1: | :+1: |
-| Mutation | :+1: | :construction: [pr](https://github.com/graph-gophers/graphql-go/pull/182) | :+1: | :+1: |
-| Subscription | :+1: | :construction: [pr](https://github.com/graph-gophers/graphql-go/pull/132) | :+1: | :+1: |
+| Mutation | :+1: | :+1: | :+1: | :+1: |
+| Subscription | :+1: | :+1: | :+1: | :+1: |
 | Type Safety | :+1: | :+1: | :no_entry: | :+1: | 
 | Type Binding | :+1: | :construction: [pr](https://github.com/graph-gophers/graphql-go/pull/194) | :no_entry: | :+1: |
-| Embedding | :+1: | :construction: [pr](https://github.com/graphql-go/graphql/pull/274) | :no_entry: | :no_entry: |
+| Embedding | :+1: | :no_entry: | :no_entry: | :no_entry: |
```

Comment:
This was re-opened in another PR, but its in the wrong column (not gophers)

Reply:
should I move it to the next column?